### PR TITLE
MMDP-403 Implements country dropdowns

### PR DESCRIPTION
**Pull request by bozicschucky**
**What does this PR do?**

Implements managing the countries dropdown.

**Description of Task to be completed?**
- [x] implement `create` country drop down
- [x] implement `update` country drop down
- [x] implement `get` country drop down
- [x] implement `delete` country drop down
- [x] consume `create` country drop down
- [x] consume `update` country drop down
- [x] consume `get` country drop down
- [x] consume `delete` country drop down

**How should this be manually tested?**

- Pull and checkout to`ft-manage-country-dropdown-MMDP-403` branch
- run `yarn dev-server` 
- route to `l0.0.0.0:3000/api/v1/dropdowns-country`
- Use postman to post some data to the endpoints

**Any background context you want to provide?**

n/a

**What are the relevant Jira issues?**

[MMDP-403](https://viisaus.atlassian.net/browse/MMDP-403)

**Screenshots (if appropriate)**

### CODE REVIEW

**Comment by p8ul** [comment]
Please ensure that your errors are descriptive to the user. The error from the server reads that the description is not allowed to be empty. That's
Screenshot 2019-03-26 at 11 33 32
 fine, but the toast message reads Validation error, not good.

**Comment by p8ul** [comment]
I believe that country names should have unique values. Please consider refactoring your functionalities and model to disallow the same values. Thank you.

**Comment by Ruiru11** [comment]
I am getting this errors in the console kindly  check them out
image

**Comment by eduhmik** [comment]
Hey @bozicschucky, nice job on the task. However, when I create a new country dropdown with others existing before, I get this error as shown in the screenshot below. Kindly fix that!

Screenshot 2019-03-26 at 15 56 14

File: client/containers/DropDowns/LGA/index.js [hunk 1]
```diff
@@ -146,6 +146,7 @@ export class LGA extends Component {
   render() {
     const { dropDowns } = this.state;
     const { states, loading } = this.props;
+    console.log(states);
```
Comment (shemogumbe) [inline client/containers/DropDowns/LGA/index.js]: Always remove debugging code from source control before pushing your code

**Comment by Ruiru11** [comment]
LGTM

**Comment by Musacoli** [comment]
LGTM

**Pull request merged**